### PR TITLE
Customized Noise Reduction, Noise Reduction is enabled based on Human Vision System.

### PR DESCRIPTION
**Pull request by FurongZhang**
Signed-off-by: FurongZhang <furong.zhang@intel.com>

### CODE REVIEW

File: va/va_vpp.h [hunk 1]
```diff
@@ -929,6 +931,33 @@ typedef struct _VAProcFilterParameterBufferTotalColorCorrection {
...
+/** \brief Custom Denoise filter parametrization. */
+typedef struct _VAProcFilterParameterBufferCustomDeNoise {
+    /** \brief Filter type. Shall be set to #VAProcFilterCustomNoiseReduction. */
+    VAProcFilterType    type;
+    /** \brief Human Vision System(HVS) Denoise Enable(true) or Disable(false) */
+    bool                hvs_enable;
+    /** \brief QP for encoding, used for HVS Denoise */
+    uint16_t            qp;
+    /**
+	 *  \brief codec format, used for HVS Denoise.
```
Comment (mypopydev) [inline va/va_vpp.h]: Format refine, please

**Comment by XinfengZhang** [comment]
now, no need to add codec profile?

**Comment by FurongZhang** [comment]
Yes. No need to add Codec Profile.

**Comment by XinfengZhang** [comment]
please delete this commit"Merge branch 'master' into denoise" .

**Comment by XinfengZhang** [comment]
please delete this commit"Merge branch 'master' into denoise" .

**Pull request merged**